website/static/js/institutionProjectSettings.js

```diff
@@ -20,10 +20,8 @@ var ViewModel = function(data) {
             url,
             {isCors: true}
         ).done(function (response) {
-            if (response.data.embeds.institutions.data.length){
-                self.availableInstitutions(response.data.embeds.institutions.data);
-                self.loading(false);
-            }
+            self.availableInstitutions(response.data.embeds.institutions.data.length ? response.data.embeds.institutions.data: []);
+            self.loading(false);
```

Comment:
Not necessarily. if the request fails, we get a message (through Raven), and the settings page never loads. The alternative is to render that there is an error, with an error message in place of whatever would go in the box.